saleor/graphql/discount/mutations/promotion/promotion_create.py

```diff
@@ -200,6 +203,27 @@ def _save_m2m(
             rule.channels.set(channels)
 
     @classmethod
-    def send_sale_notifications(cls, manager, instance):
-        # TODO: implement the notifications
-        pass
+    def send_promotion_notifications(
+        cls, manager: "PluginsManager", instance: models.Promotion
+    ):
+        cls.call_event(manager.promotion_created, instance)
+        cls.send_promotion_toggle_notification(manager, instance)
+
+    @staticmethod
+    def send_promotion_toggle_notification(
+        manager: "PluginsManager", instance: models.Promotion
+    ):
+        """Send a notify about starting or ending promotion if it hasn't been sent yet.
+
+        Send the notification when the start date is before the current date and the
+        promotion is not already finished.
+        """
+        now = datetime.now(pytz.utc)
+
+        start_date = instance.start_date
+        end_date = instance.end_date
+
+        if (start_date and start_date <= now) and (not end_date or not end_date <= now):
+            manager.promotion_toggle(instance)
```

Comment:
`send_promotion_toggle_notification` is called inside a transaction, so there is a possibility, that the task will be called on the instance that doesn't exist yet in DB.  IMO you should call it the same way as you did for `promotion_created` -> `cls.call_event`